app-arch/snappy/snappy-1.2.1.ebuild

```diff
@@ -10,9 +10,8 @@ HOMEPAGE="https://github.com/google/snappy"
 SRC_URI="https://github.com/google/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="BSD"
-# ${PV%%.*}.FUDGE where FUDGE is incremented if ABI is broken without a new SONAME
-# Please use abidiff on bumps.
-SLOT="0/${PV%%.*}.2"
+# ABI may be broken without a new SONAME. Please use abidiff on bumps.
+SLOT="0/1.2.1"
```

Comment:
https://github.com/google/snappy/issues/183

This release exists to add back compat symbols (restoring the SLOT="0/1.1" ABI)

The change in slot rebuild here is going from 1.1 where 1 is the soname and .1 is the FUDGE value, to 1.2.1 where the ABI subslot is "release tag".

That seems very wrong to me, sorry... @thesamesam, as you originally added 393270abb2073a7dea4e0d002c93bbc40d343caf, what exactly is desired here? I'm guessing use 1.1 so we can then stable this without stable arch users having to rebuild anything?

Reply:
We should go back to 1.1 if the ABI is identical to 1.1.10-r1. Using 1.2.1 isn't right.

But @winterheart, please use abidiff to make sure they really did fix the ABI to be identical to 1.1.10-r1, and on future bumps too.

Reply:
ABI changed both compared to 1.1.10 and 1.2.0. 1.2.1 subslot is chosen to distinguish bugged 1.2.0 and rebuild all dependent packages that already linked with 1.2.0.

Reply:
I can merge and change the subslot value if you guys agree. This package compiled etc was fine.

Reply:
OK for 1.1 subslot.